website/pages/en/network/developing.mdx

```diff
@@ -2,52 +2,122 @@
+1.  Create, deploy and publish subgraphs to The Graph using **Graph CLI** and [Subgraph Studio](/https://thegraph.com/studio/).
+2.  Use Graph node via **GraphQL** to query live subgraphs and power dapps.
 
-> There are certain constraints on The Graph Network, in terms of feature and network support. Only subgraphs on [supported networks](/developing/supported-networks) will earn indexing rewards, and subgraphs which fetch data from IPFS are also not eligible.
+### What is GraphQL and Graph CLI?
 
-### Deploy to Subgraph Studio
+- **GraphCLI** is a command line interface tool for building and deploying to The Graph.
+- **GraphQL** is the query language use for APIs and a runtime for executing those queries with your existing data. The Graph uses GraphQL to query subgraphs.
 
-Once defined, the subgraph can be built and deployed to [Subgraph Studio](/deploying/subgraph-studio-faqs/). Subgraph Studio is a sandbox environment which will index the deployed subgraph and make it available for rate-limited development and testing. This gives developers an opportunity to verify that their subgraph does not encounter any indexing errors, and works as expected.
+### Developer Capabilities
 
-### Publish to the Network
+- Engage with the entire subgraph lifecycle on the network, including publishing, querying and iterating on your custom subgraphs.
+- Specialize in subgraph development, building open APIs for others to utilize and build on.
+- Focus on application development by querying subgraphs deployed by other developers.
 
-When the developer is happy with their subgraph, they can publish it to The Graph Network. This is an on-chain action, which registers the subgraph so that it is discoverable by Indexers. Published subgraphs have a corresponding NFT, which is then easily transferable. The published subgraph has associated metadata, which provides other network participants with useful context and information.
+## Subgraph Specifics
 
-### Signal to Encourage Indexing
+### What are subgraphs?
 
-Published subgraphs are unlikely to be picked up by Indexers without the addition of signal. Signal is locked GRT associated with a given subgraph, which indicates to Indexers that a given subgraph will receive query volume, and also contributes to the indexing rewards available for processing it. Subgraph developers will generally add signal to their subgraph, in order to encourage indexing. Third party Curators may also signal on a given subgraph, if they deem the subgraph likely to drive query volume.
+A subgraph is a custom API built on blockchain data. It extracts data from a blockchain, process it, and stores it so that it can be easily queried via GraphQL.
```

Comment:
```suggestion
A subgraph is a custom API built on blockchain data. It extracts data from a blockchain, processes it, and stores it so that it can be easily queried via GraphQL.
```